doc/fips/SONiC-OpenSSL-FIPS-140-3-deployment.md

```diff
@@ -0,0 +1,89 @@
+# SONiC FIPS Deployment
+
+
+Table of Contents
+=================
+* [Overview](#Overview)
+* [Requirements](#Requirements)
+* [Scopes](#Scopes)
+* [SONiC Configuration for FIPS](#SONiC-Configuration-for-FIPS)
+    * [FIPS None Enforce Mode](#FIPS-None-Enforce-Mode)
+    * [FIPS Enforce Mode](#FIPS-Enforce-Mode)
+* [SONiC FIPS State](#SONiC-FIPS-State)
+
+## Revision
+
+| Rev | Date     | Author          | Change Description |
+|:---:|:--------:|:---------------:|--------------------|
+| 0.1 | 06/24/23 | Xuhui Miao  | Initial version    |
+
+## Overview
+It is for the security requirement, the FIPS 140-3 feature should be enabled for the devices in the datacenters. This design document is to enable the FIPS for enabling the FIPS for existing devices and new rollout devices, and to enable the FIPS config in the runtime in the SONiC config.
+
+## Requirements
+- Provide a way to enable/disable the FIPS in the runtime for control/management plane, without reboot data plane.

```

Comment:
>control/management plane [](http://example.com/codeflow?start=62&length=24)

Do you have a list of components impacted by enable/disable FIPS?

Reply:
We cannot list all of them, some of them have dependency on the SymCrypt, but may not use the Crypto algorithm.

Reply:
Added the components.